### ISSUE TITLE
Node Crashed on Opening on Mac OSX for the first time.

### ISSUE THREAD

**Issue by gfunksound** [ethereum/mist#243]
This appeared on the prompt and it said to open this issue:

Node type: geth
Network: main
Platform: darwin

...I0304 22:48:54.729856   26361 database.go:71] Alloted 433MB cache to /Users/XXXXX/Library/Ethereum/chaindata
I0304 22:49:18.177621   26361 database.go:71] Alloted 78MB cache to /Users/XXXXX/Library/Ethereum/dapp
I0304 22:49:18.554590   26361 backend.go:314] Protocol Versions: [63 62 61], Network Id: 1
I0304 22:49:18.827035   26361 backend.go:362] Blockchain DB Version: 3
I0304 22:49:19.965987   26361 blockchain.go:214] Last header: #1085707 [c05ab237…] TD=8287553524161034757
I0304 22:49:19.966057   26361 blockchain.go:215] Last block: #1085707 [c05ab237…] TD=8287553524161034757
I0304 22:49:19.966074   26361 blockchain.go:216] Fast block: #1085707 [c05ab237…] TD=8287553524161034757
I0304 22:49:20.201739   26361 handler.go:92] blockchain not empty, fast sync disabled

**Comment by alexvandesande** [comment]
Did you have another node running? What happens if you open it again?

**Comment by frozeman** [comment]
Maybe the error popup timeout is to short for some situations, i will increase it..

**Comment by gfunksound** [comment]
Hi, I did not have another node running. I opened it again and it seemed to
sync up fine. There is also the problem that the wallet does not remained
synced it. It will sync initially, but after a while it will fall behind
and say "last block 9 minutes or something ago" and stay there until I quit
it and open it again to sync

**Comment by frozeman** [comment]
It looks lie some network connectivity issue. Do you have peers?
@fjl any idea what this could be on MacOSX?

**Comment by fjl** [comment]
The log doesn't look like there is any issue.

**Comment by frozeman** [comment]
Its working fine for me on Mac. It seems like he has some network or peer issue.
@gfunksound Do you have a firewall or a protected router?

**Comment by gfunksound** [comment]
I don't have a firewall or protected router. The wallet worked fine with a
previous version. I forgot which one it was. It wasn't 0.5 or 0.5.1 or the
one before that. The one that worked fine was the release before the
release before 0.5, if that makes sense. After that update it has always
had this sync problem.

**Comment by adi-ads** [comment]
Node crashed while opening Ethereum-wallet for the first time as well -

Node type: geth
Network: main
Platform: darwin

...I0308 13:59:42.184184    1844 database.go:71] Alloted 433MB cache to /Users/adi/Library/Ethereum/chaindata
I0308 13:59:42.206911    1844 backend.go:760] upgrading db log bloom bins

**Comment by frozeman** [comment]
@adi-ads let you node upgrade the db as it says i the log, and wait for a while, then restart the wallet..

**Comment by gfunksound** [comment]
I do have peers and I am not behind a firewall or anything. The wallet
syncs up when it opens, but then stops downloading new blocks. It says that
I have 5 or more peers, but the time since last block keeps increasing
without the block count increasing, so it isn't downloading any new blocks.

I have updated to 0.5.2 and it still has this problem, I get the following
error in the console:

file:///sockjs/info?cb=4z2fkjxuk7 Failed to load resource:

Also it seems there is a missing mp3 file?

**Comment by clund** [comment]
Node type: geth
Network: main
Platform: darwin (Architecure x64)

...I0317 21:38:40.739994   59943 database.go:71] Alloted 433MB cache to /Users/clund/Library/Ethereum/chaindata
I0317 21:38:43.062773   59943 backend.go:685] Old database detected, upgrading...

**Comment by davronakil** [comment]
I got a similar issue. My problem is that I had etherewall running with geth already installed. Then I installed mist. It worked for a while. Then I ran geth in the terminal and tried opening Mist, and either mist or etherwall has not worked since. I get the following error on launch:

`Node type: geth
Network: main
Platform: darwin (Architecure x64)

...9:14.999164   42400 backend.go:314] Protocol Versions: [63 62 61], Network Id: 1
I0328 17:59:15.002469   42400 backend.go:362] Blockchain DB Version: 3
I0328 17:59:15.008608   42400 blockchain.go:214] Last header: #0 [d4e56740…] TD=17179869184
I0328 17:59:15.008646   42400 blockchain.go:215] Last block: #0 [d4e56740…] TD=17179869184
I0328 17:59:15.008654   42400 blockchain.go:216] Fast block: #0 [d4e56740…] TD=17179869184
I0328 17:59:15.024092   42400 cmd.go:114] Starting Geth/v1.3.5/darwin/go1.5.1
I0328 17:59:15.029041   42400 server.go:311] Starting Server
I0328 17:59:15.051756   42400 udp.go:212] Listening, enode://333f55a2dd0b1effe93b07362457dec03cd590ee9aa5e1dd19a48e7787ee6e84039a7119f9d4565dd37a501076342235461074ac56afc23c0236b0f4f808f732@[::]:30303
I0328 17:59:15.053201   42400 backend.go:526] Server started
I0328 17:59:15.053490   42400 server.go:552] Listening on [::]:30303
I0328 17:59:15.055353   42400 ipc.go:112] IPC service started (/Users/davron/Library/Ethereum/geth.ipc)
`

I have my wallet data backed up. Can i just nuke everything and start from scratch? I'll be able to restore it right? 

Any help is greatly appreciated. Thanks!
